hardware_calibration/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pykinect_azure
```

Comment:
Maybe add these requirements to the main requirements.txt or add a README detailing the calibration process as well as installation.